xbmc/Util.cpp

```diff
@@ -547,6 +548,15 @@ bool CUtil::IsPicture(const std::string& strFile)
                   CServiceBroker::GetFileExtensionProvider().GetPictureExtensions()+ "|.tbn|.dds");
 }
 
+std::string CUtil::GetSplashPath()
+{
+  std::array<std::string, 4> candidates {{ "special://home/media/Splash.jpg", "special://home/media/Splash.png", "special://xbmc/media/Splash.jpg", "special://xbmc/media/Splash.png" }};
+  auto it = std::find_if(candidates.begin(), candidates.end(), [](std::string const& file) { return XFILE::CFile::Exists(file); });
```

Comment:
pass this through CSpecialProtocol::TranslatePathConvertCase, or the better but more risky fix which is to fix CSpecialProtocolFile::TranslatePath